pkg/resolution/resolver/bundle/bundle.go

```diff
@@ -72,7 +73,12 @@ func (br *ResolvedResource) Source() *pipelinev1beta1.ConfigSource {
 // GetEntry accepts a keychain and options for the request and returns
 // either a successfully resolved bundle entry or an error.
 func GetEntry(ctx context.Context, keychain authn.Keychain, opts RequestOptions) (*ResolvedResource, error) {
-	img, err := retrieveImage(ctx, keychain, opts.Bundle)
+	uri, img, err := retrieveImage(ctx, keychain, opts.Bundle)
+	if err != nil {
+		return nil, err
```

Comment:
These errors may be a bit difficult to know where they come from on their own - you may want to wrap them with a fmt.Errorf

Reply:
Sounds good! Changed.